app/components/image-cropper.js

```diff
@@ -22,7 +22,7 @@ export default class ImageCropperComponent extends Component {
       zoomOnWheel: false,
       rotatable: false,
       toggleDragModeOnDblclick: false,
-      preview: '.image-cropper-preview',
+      // preview: '.image-cropper-preview',
```

Comment:
Can we please remove the commented code?

Reply:
done